src/AddTodoForm.jsx

```diff
@@ -1,18 +1,23 @@
+import {useState} from "react";
+
 const AddTodoForm = ({onAddTodo}) => {
+    const [todoTitle, setTodoTitle] = useState("")
 
+    const handleTitleChange = (e) => {
+        const value = e.target.value;
+        setTodoTitle(value);
+    }
     const handleAddTodo = (e) => {
         e.preventDefault();
-        const todoTitle = e.target.elements.title.value;
-        onAddTodo(todoTitle);
-        e.target.elements.title.value = ''
-
+        onAddTodo({title: todoTitle, id: Date.now()});
+        setTodoTitle(" ")
     }
 
     return (
         <>
             <form onSubmit={handleAddTodo} action="">
                 <label title="Title">
```

Comment:
I think the goal here is to have this label show as text in front of the input element: https://react.dev/reference/react-dom/components/input#providing-a-label-for-an-input.

If we put "title" inside the title property, it just shows up as a tooltip when we hover over the real text: https://www.dofactory.com/html/label/title